gems/aws-sdk-rds/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Unreleased Changes
 ------------------
 
+* Issue - Fix flaky auth_token spec caused by time zone sensitivity.
```

Comment:
Changelog isn't necessary for test files.

Reply:
OK. Fixed.